### ISSUE TITLE
Feature request: equivalent to Blender's "Frame Node" for grouping nodes

### ISSUE THREAD

**Issue by looeee** [chrisjpatty/flume#136]
[Blender's Frame Node](https://docs.blender.org/manual/en/dev/interface/controls/nodes/frame.html) is very important for grouping and handling large node graphs. Has this feature been considered here yet? 

![](https://docs.blender.org/manual/en/dev/_images/interface_controls_nodes_frame_example.png)

**Comment by chrisjpatty** [comment]
Have you played with the built-in "Comment" node? Comment nodes always sit below other nodes and can be resized like blender's frame nodes. Is that sufficient for what you're thinking or do frame nodes have additional features you need?

**Comment by looeee** [comment]
Yes, the difference is the Frame node is for quick grouping of nodes and once they are in a frame you can move the group around. 

https://user-images.githubusercontent.com/5307958/155838687-878d1cd6-9eae-4e99-a324-cd0224be609e.mp4

**Comment by rohanshetty-dev** [comment]
I want to add this feature in my project. Is it implemented yet? If not then, how should I approach, would like to add this feature. 
Thanks!
